_includes/homepage/homepage-hero-band.html

```diff
@@ -35,7 +35,7 @@ <h4>Kubernetes-Native Experience</h4>
         <p>
           <ul>
             <li>For example kubectl get kafka</li>
-            <li>Operator Based (<a href="{{site.baseurl}}/docs/overview/latest/">What is an operator</a> and why should I use them?)</li>
+            <li>Operator Based (<a href="https://kubernetes.io/docs/concepts/extend-kubernetes/operator/">What is an operator</a> and why should I use them?)</li>
```

Comment:
I rasied it already 10 times, but I will do it again because I'm persistent. I think we should remove the `and why should I use them` part. It makes this point to be on two lines which looks bad to me on the website and I do not think it really adds any value.

Reply:
and I forgot it 11 times ;-) Just fixed it.

Reply:
Should we keep the `?` ... `What is an operator?`?

Reply:
Done